### ISSUE TITLE
Share count for google plus

### ISSUE THREAD

**Issue by codenameyau** [newsela/sharing-metrics#1]
All of the articles I read either require an API key or a web scraper. I'm gonna go with the web scraper.
http://blog.sriraman.in/url-share-count-facebook-twitter/

Scape this url `https://plusone.google.com/_/+1/fastbutton?url=` and look for '#aggregateCount'

**Comment by antinescience** [comment]
Is that going to be brittle down the line, though? It might be worth allowing for the API route if an env var with an API key exists and then falling back if one doesn't.

**Comment by codenameyau** [comment]
Actually, the google plus scraper is not a great idea. The button doesn't give an accurate number of shares. For example, this is what is returned.

```html
<div id="aggregateCount" class="Oy">1k</div>
```

**Comment by antinescience** [comment]
Yeah. I'd say just look for an API key for it somewhere. Environment variable, config file, etc.

**Comment by codenameyau** [comment]
Did some research. Brace yourself.

I got an google API key and followed the docs here. Didn't work. Their API is private. http://stackoverflow.com/questions/28203030/google-plus-api-count-call-failing

I found a russian website that will hook us up with the counts. Everyone seems to be using it.
http://jsfiddle.net/4gvghj51/

Current scraper method.
http://stackoverflow.com/a/15449276/5270126

Outrage from the rest of the community because there's still no API.
https://code.google.com/p/google-plus-platform/issues/detail?id=57

**Comment by codenameyau** [comment]
FOR MOTHER RUSSIA

**Comment by codenameyau** [comment]
Everything is broken including http://jsfiddle.net/4gvghj51/

This is also broken.
http://stackoverflow.com/questions/29016599/get-google-plus-url-share-count
